### PR TITLE
Coroner Update: Pickle-Eating Morbid Weirdo Obsessed with Death and Perfectionism

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

This PR introduces a whole bunch of Coroner and Morbid related content.

Firstly, Morbid is now a mind trait, and specifically, coroners start with it.

Coroners also have a liver trait that allows them to heal toxins (very slowly) from eating pickles and drinking pickle juice. They also can...drink formaldehyde. I guess. Dissections is thirsty work.

Coroners gain a whole set of special tools specifically for use in any surgeries marked as interests of the Morbid. This is determined by the ``surgery_flag`` called ``SURGERY_MORBID_CURIOSITY``. Currently, these surgeries are included;

dissections, autospies, revival surgery, plastic surgery, organ/feature manipulations, amputations

To fit the theme, TRAIT_MORBID also applies the reduction to eye snatchers.

While using their special tools, and the surgery is a morbid curiosity, the coroner/anyone who is morbid gains a 30% speed boost! This stacks with the dissection speed boost. Otherwise, the tools are just regular tools with a special name (though the scalpel is better at killing undead, because, you know, you're watching over the dead).

The coroner's special medkit, which is the only one you can get in a round, can fit their autopsy scanners and tools. Anything that comes standard with their kit can go back into it.

Anyone who is morbid can safely retrieve the secrets of the elephant graveyard. The serrated shovel, notably, is a much better tool and notably better at killing organics, but not inorganics (like the dead).

(Gives roboticists secure morgue access during skeleton crew pop totals)

## Why It's Good For The Game

I really loved the coroner role and what it had going conceptually. And well, I can't help but come back over and start adding some QOL updates and some content to flesh it out and encourage a specific role for the coroner to play. 

That utterly deranged freak in the morgue who seems very much intent on playing god and artist in equal measure.

So, to encourage these players to look for and perform specific surgeries, they are penalized for the more typical work done by medical staff, but they are handsomely rewarded for doing some of the more unique surgeries we have. While this expands their role a little beyond just tending the dead, I think this is perfectly fine as a kind of special side content additive. 

This is also very true if we want to make sure coroners are sought out for their ability to contribute to medical's success via dissections. But I think having people look to them for frankensteining is also very thematic. Especially if they get more than they bargained for by doing so.

I thought them loving pickle juice was funny because they're pickling people all round. **This has no relation to any sentient pickles, fuck off with that shit.**

Also I had like maybe 4 or 5 people ask me to do this since the vorpal scythe pr so I knew this was rather anticipated.

## Changelog
:cl:
add: Coroners are now Morbid!
add: Coroners come with a series of special tools that are especially good at performing certain surgeries if used by a Morbid individual (which the coroner happens to be). They are found in the coroner's medkit.
add: The sorts of surgeries they enjoy are; dissections, autospies, revival surgery, plastic surgery, organ/feature manipulations, amputations. Also eyesnatching. 
add: Coroners therefore hate; tending the wounds of the living, defibrillation and CPR. Why waste so much effort on breathers if you can't even carve them up a bit first?
add: Coroners love pickles and pickle juice.
add: The coroner can finally put their autopsy scanner into their special medkit. Not the compact one, though.
add: The elephant graveyard is safer to plunder for the morbid individual. The rewards from the graveyard are now also slightly more lucrative...if you don't care about being cursed, that is.
balance: Roboticists get secure morgue access during skeleton shifts
/:cl:

### CODE REVIEW

**Comment by Sealed101** [comment]
bone shovel believers!!!

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -269,7 +269,7 @@
 		/obj/item/clothing/glasses/regular = 2,
 		/obj/item/clothing/glasses/regular/jamjar = 1,
 		/obj/item/storage/bag/books = 1,
-		/obj/item/radio/headset/headset_srv = 2,
+		/obj/item/radio/headset/headset_srvmed = 2,
```
Comment (JohnFulpWillard) [inline code/modules/vending/wardrobes.dm:272]: ?

Comment (necromanceranne) [reply code/modules/vending/wardrobes.dm:272]: These are in the vending machines for replacements/job swaps. You can buy the normal med headset, pop out the encryption key out, and insert it into the service headset key and get the same results.

Coroners get a discount for both vendors, so why make the process of gearing up more difficult than necessary? Or make it harder for coroners to acquire roundstart equipment replacements?

Comment (necromanceranne) [reply code/modules/vending/wardrobes.dm:272]: Oh this is the wrong vendor. My b.

**Comment by CRITAWAKETS** [comment]
If you want to update coroner, could you also do something about the fact that their office doesn't have any skeleton crew access to medical doctors?

File: code/modules/mining/equipment/mining_tools.dm [hunk 2]
```diff
@@ -164,18 +164,24 @@
...
-	force = 15
+	force = 10
 	throwforce = 12
 	w_class = WEIGHT_CLASS_NORMAL
-	toolspeed = 0.7
+	toolspeed = 0.3
 	attack_verb_continuous = list("slashes", "impales", "stabs", "slices")
 	attack_verb_simple = list("slash", "impale", "stab", "slice")
 	sharpness = SHARP_EDGED
+	item_flags = CRUEL_IMPLEMENT
```
Comment (carlarctg) [inline code/modules/mining/equipment/mining_tools.dm:180]: Would probably be a good idea to add an examine string for morbids to show that this and other items are cruel implements

**Comment by carlarctg** [comment]
The cruel scalpel feels like it should have a black TIP, not body. Maybe even make it an obsidian scalpel for the cool flavor?  Those are used IRL for people with severe metal allergies and they're pretty morbidlike aestheticore.

The morbid cautery is too similar to the advanced cautery. I would give it a black strip and a black tip on the normal white sprite

**Comment by ATH1909** [comment]
maybe you should add prosthetic limb attachment to the list of morbid surgeries, since it's used to stitch new limbs onto the people you've amputated?

10% also seems like... a very small buff?

**Review by Wallemations — APPROVED** [review]

**Comment by nukularpower** [comment]
its might be a bit out of scope but it would be cool to see sleepers/cloners coem back as researchable techs locked behind coroner/medbay work and experiments

**Comment by JohnFulpWillard** [comment]
Yeah a little bit

File: code/modules/mining/equipment/mining_tools.dm (hunk 2 above)
Comment (necromanceranne) [reply code/modules/mining/equipment/mining_tools.dm:180]: Very fair

**Comment by EricZilla** [comment]
How much of a penalty do you get for doing first aid as a coroner after this? I get the idea is to make them hate healing but they ARE part of medical

**Comment by necromanceranne** [comment]
It's a mood penalty for about 2 minutes. If the person is dead, they won't suffer the mood penalty.

**Comment by EricZilla** [comment]
Yeah that sounds pretty good then

**Comment by Time-Green** [comment]
Just ping me when you feel this is done

**Comment by RikuTheKiller** [comment]
A 10% boost? That's... pointless. Even sterilizine is way, way stronger than that and yet I can't even tell the difference.
I'd much rather have something like a 50% boost even if it'd require harder balancing.

**Comment by JohnFulpWillard** [comment]
> A 10% boost? That's... pointless. Even sterilizine is way, way stronger than that and yet I can't even tell the difference.
I'd much rather have something like a 50% boost even if it'd require harder balancing.

This stacks with sterilizine and dissected bodies, which is together ~20% boost (i know dissected bodies get 10% idk about sterilizine).

**Comment by necromanceranne** [comment]
> > A 10% boost? That's... pointless. Even sterilizine is way, way stronger than that and yet I can't even tell the difference.
> > I'd much rather have something like a 50% boost even if it'd require harder balancing.
> 
> This stacks with sterilizine and dissected bodies, which is together ~20% boost (i know dissected bodies get 10% idk about sterilizine).

I made it 30% just because it's very specific and more superficial surgeries compared to our main surgeries (tend wounds)

**Comment by JohnFulpWillard** [comment]
> I made it 30% just because it's very specific and more superficial surgeries compared to our main surgeries (tend wounds)

Coroners start with sterelizine and the only object that can dissect, meaning they are roundstart able to stack up to 50% surgery speed on bodies. Is this something we want?

**Pull request merged**